### ISSUE TITLE
install issue

### ISSUE THREAD

**Issue by ssdonelson** [judsonmitchell/ClinicCases#39]
the installation stalls at this point .......looking for suggestions or guidance

Thanks for trying out ClinicCases. Let's get started with the install.

Your php version is 8.1.2. Good.

People and Uploads directories are writable. Good.

The config file is writable. Good.

**Comment by sudolewgs** [comment]
I am also having this issue. Have you found anything out yet?

**Comment by ssdonelson** [comment]
No they do not reply so I am building an application in python-django
framework

**Comment by test2a** [comment]
What is the exact issue ? Does it say folder not accessible?

**Comment by sudolewgs** [comment]
It says the same as your issue where everything is writable but nothing happens after that. I can't tell if I am missing a step.

**Comment by ssdonelson** [comment]
I have not received an answer yet. I am writing new program in python
django framework which I anticipate will be ready after new year

**Comment by test2a** [comment]
@ssdonelson that is unfortunate. i was able to solve one of the issues by going to php 6 or even 5. also, your program, will it be open source? is it on github? can we test it?

**Comment by ssdonelson** [comment]
I am still working on it and will release it soon in github possibly

**Comment by ravenswd** [comment]
Has anyone done any work on upgrading ClinicCases to php 8?

**Comment by test2a** [comment]
@ravenswd i dont think so. the current overhaul being done currently is a UI overhaul only.

if you have the technical chops to do the upgrade of version8 branch to latest php, lets start working on it and maybe we can fix this... what do you think

**Comment by ravenswd** [comment]
@test2a I just found a hosting service that supports PHP 5.6 and set-up a website there. It's working so far.

It's the dumb way to fix it, but if the dumb way works then it isn't dumb.

**Comment by test2a** [comment]
@ravenswd did you have to do that mysql strict mode?

something to test. do ical/calendar sync on your phone and see if that works properly.

btw, did you use version 8 or the existing install?
